components/support/rc_crypto/nss/nss_build_common/src/lib.rs

```diff
@@ -78,6 +78,8 @@ fn link_nss_libs(kind: LinkingKind) {
     for lib in libs {
         println!("cargo:rustc-link-lib={}={}", kind_str, lib);
     }
+    // Link against C++ stdlib (for mozpkix)
+    println!("cargo:rustc-link-lib=stdc++");
```

Comment:
Nice find. I would like us to be using a more structured approach to the build here such as the [`cc` crate](https://github.com/alexcrichton/cc-rs) but that's well beyond the scope of this PR.

One thing to watch out for here, is apparently the name of this may change on different platforms, as described [here](https://github.com/alexcrichton/cc-rs#c-support). We'll have to do some manual testing on Android to see if any tweaks are needed.

Reply:
On my Mac, I had to change this to `println!("cargo:rustc-link-lib=c++");` to get it to compile successfully. (The tests fail in something about splitting the pem, but that seems unlikely to be related to c++ linking issues).